src/transformers/models/falcon/modeling_falcon.py

```diff
@@ -67,6 +78,19 @@ def rotate_half(x):
     return torch.cat((-x2, x1), dim=-1)
 
 
+# Copied from transformers.models.llama.modeling_llama._get_unpad_data
+def _get_unpad_data(attention_mask):
```

Comment:
and the PR and explanation on why we may want to keep this method here https://github.com/younesbelkada/transformers/pull/4